examples/simple/readme.md

```diff
@@ -0,0 +1,56 @@
+# Catalyst server logs configuration
+
+The follow diagram explain how our catalyst server manage the logs
+
+![threads-diagram drawio](https://user-images.githubusercontent.com/88118994/149195822-de5d33ad-f29f-48ff-840b-ce2fb41eb08a.png)
+
+## [Pino-pretty](https://github.com/pinojs/pino-pretty) logs for development mode:
+
+install pino-pretty as a development dependency.
+
+` npm install -D pino-pretty `
+
+manifest.json
+```
+    "hapi-pino": {
+        "plugin": "require:hapi-pino",
+        "options": {
+            "$filter": "env.NODE_ENV",
+            "$default": {},
+            "development": {
+                "transport": {
+                    "target": "pino-pretty",
+                    "options": {
+                        "colorize": true,
+                        "translateTime": true
+                    }
+                }
+            }      
+        }
+    }
+```
+
+## [Pino-pretty](https://github.com/pinojs/pino/blob/master/docs/pretty.md) logs for production mode(legacy transport):
+
+Install Pino-pretty globally
+
+`npm install -g pino-pretty`
```

Comment:
I don't think we want to encourage installing globally. The code below will work even if installed locally, but with a change of path, `./node_modules/.bin/pino-pretty`